admin_manual/installation/source_installation.rst

```diff
@@ -2,47 +2,177 @@
+     OnlyOffice, Talk and so on. Full feature list available 
+     `here <https://github.com/nextcloud/nextcloudpi?tab=readme-ov-file#features>`_.
+   - `Official Documentation <https://github.com/nextcloud/nextcloudpi>`_ (**Note**: see Official Documentation 
+     for NCP specific installation and usage details).
+* Nextcloud Community Docker Image
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - Two editions of the image are provided for building different types of Nextcloud stacks: the Apache image 
+     contains a full Nextcloud installation including an Apache web server. The second option is an FPM 
+     installation and runs a FastCGI process that serves your Nextcloud installation. The standard images are
+     Debian-based, but Alpine variants are offered as well.
+   - This image is designed for expert use and intended to be used in a micro-service environment where you can
+     incorporate as a building block in your own Nextcloud stack (i.e. you will not need to install/update 
+     Nextcloud, but you will need to supply your preferred web, reverse proxy, HTTPS termination, database and 
+     other desired supplementary services).
+   - Full Nextcloud stacks, incorporating the image, are provided in the form of example Docker Compose files 
+     and Dockerfile variations. Full feature list available `here <https://github.com/nextcloud/docker/>`_.
+   - `Official Documentation <https://github.com/nextcloud/docker/>`_ (**Note**: see Official Documentation 
+     for Image specific installation and usage details).
+* Nextcloud Web Installer
+   - The Web Installer is an easy way to install Nextcloud Server in a shared / managed web space (e.g. shared 
+     hosting) if you don't have access to the command line and if your environment meets the requirements.
+   - Checks for essential PHP dependencies, downloads the Nextcloud Archive file from the official server, and 
+     unpacks it with the right permissions then directs you to the Nextcloud Server Setup Wizard.
+   - `Official Documentation <https://github.com/nextcloud/web-installer/>`_
+* One Click Signup
+   - An alternative to providing your own installation environment and installing and keeping a Nextcloud Server 
+     up-to-date yourself.
+   - Hosting offered by independent Nextcloud providers from all over the world.
+   - Each has committed to providing you 2+ GB of storage on a 100% free single user account, with all the basic 
+     Nextcloud apps. 
+   - Through Simple Signup, getting started is made as easy as possible. You can with the suggested default provider 
+     (based on your location) or choose another.
+   - To sign-up click *Sign up with a provider* `here <https://nextcloud.com/install/#one-click-signup>`_.
+* Nextcloud Enterprise
```

Comment:
I would perhaps add the point, to integrate Nextcloud in existing system infrastructure (specific user/storage/authentication backends etc.), which might be more difficult and require more experience than standalone setups.